scripts/build.sh

```diff
@@ -13,6 +12,7 @@ VERSION=debug
 TARGET="$(uname -m)-unknown-linux-gnu"
 CRATE="youki"
 RUNTIMETEST_TARGET="$ROOT/runtimetest-target"
+features=""
```

Comment:
Why do we need this change?

Reply:
Because `features` as a variable is not set nor has a default value. Bash complaints when I turned on the `-u`, because `features` is unbounded if no opts is specified. Later in the script, we test for features as a unbounded variable with if [ -n ], but we should be explicit with `features = ""` so that it is bounded. There is a more strict set of bash options I've recently started to put in all my bash scripts:

```bash
    #!/usr/bin/env bash
    set -euo pipefail
```